spec/controllers/catalog_controller_spec.rb

```diff
@@ -18,6 +18,31 @@
     allow_any_instance_of(ApplicationController).to receive(:fetch_path)
   end
 
+  it "checks method st_delete" do
+    login_as admin_user
+    allow(User).to receive(:current_user).and_return(admin_user)
+    allow(controller).to receive(:checked_or_params).and_return(ServiceTemplate.all.ids)
+    controller.instance_variable_set(:@_params, {}) # affects params in controller
+    allow(controller).to receive(:find_checked_items).and_return(ServiceTemplate.all.ids)
+    allow(controller).to receive(:replace_right_cell)
+    expect(controller).to receive(:process_sts).with(ServiceTemplate.all.ids, "destroy")
+    controller.send(:st_delete)
+  end
+
+  it "checks method x_edit_tags_reset" do
+    login_as admin_user
+    allow(User).to receive(:current_user).and_return(admin_user)
+    allow(controller).to receive(:checked_or_params).and_return(ServiceTemplate.all.ids)
+    controller.instance_variable_set(:@_params, {}) # affects params in controller
+    controller.instance_variable_set(:@sb, :action => nil)
+    allow(controller).to receive(:checked_or_params).and_return(ServiceTemplate.all.ids)
+    allow(controller).to receive(:find_checked_items).and_return(ServiceTemplate.all.ids)
```

Comment:
this line shouldn't be necessary, as you're already mocking checked_or_params.

(same as https://github.com/ManageIQ/manageiq-ui-classic/pull/3891/files#r187269347)